dotnet/docs/EXPERIMENTS.md

```diff
@@ -76,3 +77,6 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 | | | | | | | |
 | SKEXP0101 | Experiment with Assistants | | | | | |
 | SKEXP0101 | Experiment with Flow Orchestration | | | | | |
+| | | | | | | |
+| SKEXP0110 | Agent Framework | | | | | |
+| SKEXP0111 | ChatCompletionAgent | | | | | |
```

Comment:
We recently moved away from the fine-grained experimental IDs. SKEXP0110 for all agents should be aligned with the new convention.

Reply:
I just learned that from Mark as well.  Keeps thing more simple, great!